src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,65 @@
+
+
+
+::ng-deep {
```

Comment:
This will affect all menus in the whole app.  If that's what you want it shouldn't be in this component, if it's not you should use :host::ng-deep to apply it only to this component (and children).